geomesa-convert/geomesa-convert-json/src/main/scala/org/locationtech/geomesa/convert/json/JsonSimpleFeatureConverter.scala

```diff
@@ -40,6 +46,15 @@ class JsonSimpleFeatureConverter(jsonConfig: Configuration,
       Array[Any](o)
     }.toSeq
 
+  // TODO - currently a hack to make things work
+  override def process(is: InputStream, ec: EvaluationContext = createEvaluationContext()): Iterator[SimpleFeature] =
+    lineMode match {
+      case LineMode.Single =>
+        processInput(Source.fromInputStream(is).getLines(), ec)
```

Comment:
should we specify the encoding here? probably better to explicitly use UTF-8 than rely on platform default. Same with IOUtils, below.